app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,7 @@
+
+<div id="recipes">
+  <div class="foods-index-button"><%= link_to "Add recipe", new_recipe_path, class: 'btn btn-primary' %></div>
+  <% @recipes.each do |recipe| %>
+    <%= render recipe %>
+  <% end %>
+</div>
```

Comment:
![image](https://github.com/zbahati/recipeApp/assets/93039359/4d4c53a2-a0b4-4559-8ad2-b62686a5bf26)

- [ ] Great work so far on making the Recipes index page but it would be nice if you align the contents in the recipe cards because you can see the names are not aligned well and the description too :fearful: Kindly fix this issue to make your app look professional to follow the project requirements :+1:

Reply:
i aligned the cards data